client/aliases.go

```diff
@@ -81,6 +81,8 @@ func (client *Client) Unalias(aliasOrSnap string) (changeID string, err error) {
 type AliasStatus struct {
 	App    string `json:"app,omitempty"`
 	Status string `json:"status,omitempty"`
```

Comment:
I do think we want to keep Status, because if we evolve things further like having separate/multiple sets of aliases per snap, as we were thinking at some point, exposing too much the internal current logic will get messy to migrate through